### PR TITLE
[sample] Support multi client for bug fix

### LINKED ISSUE #1190: [samples] ble related samples should be updated to support connections with multiple clients
### Description

BLE related samples are not suitable after ble support multiple clients. It will cause a lot of issues.
 
### Test Code

[WebBluetoothGroveLcdDemo.js](https://github.com/01org/zephyr.js/blob/master/samples/WebBluetoothGroveLcdDemo.js); [SensorBLEDemo.js](https://github.com/01org/zephyr.js/blob/master/samples/SensorBLEDemo.js)
 
### Steps to Reproduction

1. `make samples/WebBluetoothGroveLcdDemo.js ROM=256`
2. `make dfu`
3. Connect A101 with Nexus 5 via nRF app.
### Actual Result

1. [WebBluetoothGroveLcdDemo.js](https://github.com/01org/zephyr.js/blob/master/samples/WebBluetoothGroveLcdDemo.js);
Public client(78:31:c1:bc:1d:11) always tries to connect, then disconnect with ble server. And word on GroveLCD will disappear when public client connecting and disconnecting.
selection_015
img_20170523_152148

2. [SensorBLEDemo.js](https://github.com/01org/zephyr.js/blob/master/samples/SensorBLEDemo.js)
Public client(78:31:c1:bc:1d:11) always tries to connect, then disconnect with ble server. And Error (`error: SensorError - start failed`) occurs when the second client disconnected.
selection_012
 
### Expected Result

Samples support connections with multiple clients. 

### Test Builds

Branch | Commit Id | Target Device | Test Date | Result
---|---|---|---|---
master | 82ef9f8 | Arduino 101 | May 23, 2017 | Fail

### PR DESCRIPTION
**Pull request by cuiyanx**
Fixed #1190

Signed-off-by: Cui Yan <yanx.cui@intel.com>

### CODE REVIEW

File: samples/WebBluetoothGroveLcdDemo.js [hunk 1]
```diff
@@ -186,7 +186,6 @@ ble.on('advertisingStart', function(error) {
 
 ble.on('accept', function(clientAddress) {
     console.log("Client connected: " + clientAddress);
-    glcd.clear();
```
Comment (grgustaf) [inline samples/WebBluetoothGroveLcdDemo.js]: Why this change?

Comment (cuiyanx) [reply samples/WebBluetoothGroveLcdDemo.js]: @grgustaf If not remove glcd.clear() function, the Grove LCD will be cleared when another BLE client connected, and the Grove LCD will be not displayed until value updated.

**Comment by grgustaf** [comment]
@jimmy-huang please check this one since it's sensors + grove lcd

**Comment by jimmy-huang** [comment]
+1

**Pull request merged**